json/filter.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I think having this file as a json is OK for now, but heads up that we will probably be retrieving these filters from the database at some point. 

You can move this file to the `constants` folder.

Reply:
JK. Make this a `.dart` file and save the filters as a list. 😆